Sources/ContentView.swift

```diff
@@ -5,25 +5,32 @@ struct ContentView: View {
     @State
     var repositories = [Repository]()
     @State
-    private var searchTerm: String = "php"
+    private var searchTerm: String = ""
+    @State
```

Comment:
Instead of isError rename the variable somehow different. Variable needs to be named to be clear and self-explainatory.
Good example would be errorPresented or alertPresented or combination of those two errorAlertPresented